microsoft-edge/toc.yml

```diff
@@ -1336,6 +1336,11 @@
           href: web-platform/password-reveal.md
           displayName: 
 
+        - name: Updates to Region information exposed to web content
+          href: web-platform/os-regional-settings.md
+          displayName: 
```

Comment:
displayName: = comma-delimited list of Index lookup keywords user can type in the _ to find this article.  We should leverage this.  what Index lookup keywords can we add to make this article appear when user types that keyword into upper left at dmc?  For the "Filter by title" multipurpose text box at https://docs.microsoft.com/en-us/microsoft-edge/developer/

Reply:
Some keyword ideas: globalization, language, region, limited, intl, date, time, format

Reply:
(Added a commit with those recommended values; feel free to edit as you wish, of course.)